.gitignore

```diff
@@ -41,6 +41,12 @@ objfiles.txt
 .pydevproject
 .cproject
 .settings
+.classpath
+
+# IDEA Project
+*.iml
+.idea
```

Comment:
this removes IDEA editor artifacts from the list of potential GIT changes as those artifacts should never be committed.